domain/model/state/state.go

```diff
@@ -45,6 +92,7 @@ func NewState(
 func (s *State) Create(
```

Comment:
For discussion - we should avoid "naked" Create() and Get() etc, even in state package. IMHO.